modules/aggregation-layers/src/heatmap-layer/heatmap-layer.ts

```diff
@@ -393,7 +392,7 @@ export default class HeatmapLayer<
     const {gl} = this.context;
     const {weightsTextureSize} = this.props;
 
-    const textureSize = Math.min(weightsTextureSize, getParameters(gl, gl.MAX_TEXTURE_SIZE));
+    const textureSize = Math.min(weightsTextureSize, gl.getParameter(gl.MAX_TEXTURE_SIZE));
```

Comment:
This PR does not move WebGLContext to Device. That is too big of a change IMO, unless it's no longer possible to support non-device usage.

Reply:
As you can see in the typings I pasted in, the legacy function prototype still supports calling with `WebGLRenderingContext` instead of device.

Devices will be created under the hood regardless and associated with your WebGL context.